pattern-library/sass/patterns/_forms.scss

```diff
@@ -260,12 +260,15 @@ $input-width: (
 
     // Input fields immediately followed by a button (ie, search boxes) should eliminate the spacing
     // between them and adopt matching styling/sizing to emphasize that they are connected.
-    &[type=search] + button {
+    + button {
         border-top-left-radius: 0;
         border-bottom-left-radius: 0;
         font-size: font-size(base);
         margin-left: (-1 * $baseline / 4);
     }
+    &::-webkit-search-cancel-button {
+        display: none;
```

Comment:
Is this necessary, if `input[type="search"]` already has `-webkit-appearance: textfield`?